cloud-regionsrv-client.spec

```diff
@@ -131,6 +133,7 @@ Enable/Disable Guest Registration for Microsoft Azure
 %setup -q
 %if 0%{?suse_version} == 1315
 %patch0
+%patch1
```

Comment:
The `patch#` syntax is deprecated, please switch to `%patch -P 0` and `%patch -P 1`

Reply:
ok

Reply:
done